string position
---
float32 point_x
float32 point_y
float32 point_z
